### ISSUE TITLE
Be able to see both input and output tokens?

### ISSUE THREAD

**Issue by T-Lind** [langchain-ai/langsmith-cookbook#180]
It would be nice to see the number of input and output tokens separately.

**Comment by hinthornw** [comment]
Are you asking specifically for this in the table view? 

If you hover, you see the breakdown:
<img width="292" alt="image" src="https://github.com/langchain-ai/langsmith-cookbook/assets/13333726/f914bae2-8503-4fd1-84fb-f413e12ab015">

**Comment by T-Lind** [comment]
> Are you asking specifically for this in the table view?
> 
> If you hover, you see the breakdown: <img alt="image" width="292" src="https://private-user-images.githubusercontent.com/13333726/304220644-f914bae2-8503-4fd1-84fb-f413e12ab015.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5Mjg5MjUsIm5iZiI6MTcwNzkyODYyNSwicGF0aCI6Ii8xMzMzMzcyNi8zMDQyMjA2NDQtZjkxNGJhZTItODUwMy00ZmQxLTg0ZmItZjQxM2UxMmFiMDE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDE2MzcwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1OTAxZDMwNjk4OTYwODEzMGNiNjhiNWVhNDI1MmNlZGU1OTE0ZmM3YjUzYjI1OTFiMmE0NmZmOGRkNzYxOTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hi0DGUlk0tih8nphmQ9sMAh-FoLCJA1tB6ksrbanpBA">

Correct, because pricing is usually separate between the two, so it would be helpful. Also median input and median output separated would be nice.

**Comment by T-Lind** [comment]
Being able to see in aggregate the total number of prompt and total number of completion tokens would be very nice.
